js-foundations.js

```diff
@@ -32,19 +55,35 @@ var testString2 = "academy"
 var testString3 = "sandiego"
 
 // 4a. Write a function called alphabetSoup that takes one argument and returns a string with all the letters in alphabetical order. Use each of the varibales as test cases. Expected output: "aelnr" "aacdemy" "adeginos"
-
-
+var tS1 = testString1.split("").sort()
+var tS2 = testString2.split("").sort()
+var tS3 = testString3.split("").sort()
+const alphabetSoup = (x) =>{
+    return x
+}
+console.log(alphabetSoup(tS1).join(""))
+console.log(alphabetSoup(tS2).join(""))
+console.log(alphabetSoup(tS3).join(""))
 
```

Comment:
You can reduce the code you write by passing one of the string.split("").sort() into your function. Remember a function should be a reusable piece of code that can be called many times.